terms.html

```diff
@@ -118,6 +118,13 @@
   <dd>
 A program, such as a storage vault or personal <a>verifiable credential</a>
 wallet, that stores and protects access to <a>holders'</a>
+<a>verifiable credentials</a>.
+  </dd>
+  <dt><dfn data-lt="resources">resource</dfn></dt>
+  <dd>
+Anything, including documents, people, physical objects, and abstract concepts,
+that information can be expressed about (see [RDF primer](https://www.w3.org/TR/rdf11-primer/#introduction)).
+<a>Entities</a> are resources, but the not every resources is an <a>entity</a>.
```

Comment:
```suggestion
Every <a>Entities</a> can be expressed as a  <a>resources</a>, but the not every resources is an <a>entity</a>.
```

Reply:
even with the correction, I am not sure I agree.

Reply:
see the comments above.

Reply:
```suggestion
As [noted in the Introduction](https://www.w3.org/TR/rdf11-concepts/#resources-and-statements) of [[[RDF11-CONCEPTS]]] [[RDF11-CONCEPTS]], the term <a>entity</a> is synonymous with <a>resource</a>. In other words, every <a>entity</a> is a <a>resource</a>, and every <a>resource</a> is an <a>entity</a>
```

Reply:
How can this be resolved without changing the existing text in any way? Particularly given the text and reference I provided above? Even if my text is not fully accepted, there should be some actual response thereto, not a silent "resolved!"